examples/audio/audio_looping.rs

```diff
@@ -0,0 +1,49 @@
+use bevy::asset::LoadState;
+use bevy::prelude::*;
+
+/// This example illustrates how to load and play an audio file on repeat.
```

Comment:
This probably belongs at the top of the file, rather than attached to `OverworldTheme`.